samples/03_org_administrators/clone_storymap_version2.ipynb

```diff
@@ -0,0 +1,709 @@
+{
```

Comment:
<p>Reduce heading level, also, can we title this as <strong>Step 2: Get the StoryMap Item's...</strong></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/797/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Done</p>